### DISCUSSION TITLE
Hyperlinks in profile config not working.

### DISCUSSION THREAD

**Discussion by p145085** [community/community#40248] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I've tried multiple formats. Wanting to get a custom text hyperlinked to a URL. Simple enough, right? Doesn't seem to be working for me. What am I doing wrong?

**Comment by radar-blip** [comment, marked as answer]
Try like this:
`[Link Title](https://github.com/)`

That will get you this:
[Link Title](https://github.com/)

Here's another off your profile: `[Tor Foundation](https://www.torproject.org)`
Which gives you this: [Tor Foundation](https://www.torproject.org)

**Reply by p145085** [reply]
Thanks for suggesting this as I thought this was the correct format but I've already tried this variant and it did not work.

**Reply by p145085** [reply]
If you'll have another look at my profile you can see I've updated the links to the specified format and you can see that they're still not behaving properly.

**Reply by radar-blip** [reply]
Ok. Me thinks the indentation is causing the problem. 
Try removing indentation. 👍🏻

**Reply by p145085** [reply]
I guess it was the indentation, shame, I liked the indentation. I resorted to tables instead.

**Reply by radar-blip** [reply]
I was going to mention using tables, but I figured I'd leave formatting up to you. 
Anyways, It looks fine.
